android/quest/src/androidTest/java/org/smartregister/fhircore/quest/ui/shared/components/ActionableButtonTest.kt

```diff
@@ -36,21 +35,78 @@ import org.smartregister.fhircore.engine.configuration.workflow.ActionTrigger
 import org.smartregister.fhircore.engine.configuration.workflow.ApplicationWorkflow
 import org.smartregister.fhircore.engine.domain.model.ActionConfig
 import org.smartregister.fhircore.engine.domain.model.ResourceData
+import org.smartregister.fhircore.engine.domain.model.ServiceStatus
 
 class ActionableButtonTest {
-
   @get:Rule val composeRule = createComposeRule()
-
   private val navController = mockk<NavController>(relaxed = true, relaxUnitFun = true)
 
-  @Before
-  fun init() {
+  @Test
+  fun testActionableButtonRendersAncClickWorksCorrectlyWithStatusDue() {
+    setContent(ServiceStatus.DUE.name)
+
+    composeRule
+      .onNodeWithText("Button Text", useUnmergedTree = true)
+      .assertExists()
+      .assertIsDisplayed()
```

Comment:
this is still a little weak, more of a code path test vs a results test, I didn't see an obvious way to test text color